packages/lesswrong/components/Layout.tsx

```diff
@@ -287,6 +270,28 @@ class Layout extends PureComponent<LayoutProps,LayoutState> {
   }
 }
 
+declare global {
```

Comment:
Why do we have to declare this a global? The interface is only used within this file, if I am not confused.

Reply:
`LayoutProps` and `LayoutState` don't have to be, but the `ComponentTypes` interface does.